src/dvc_objects/fs/generic.py

```diff
@@ -168,7 +169,8 @@ def test_links(
                 if _test_link(link, from_fs, from_file, to_fs, to_file):
                     ret.append(link)
             finally:
-                to_fs.remove(to_file)
+                with suppress(FileNotFoundError):
+                    to_fs.remove(to_file)
```

Comment:
Maybe it's worth specializing this function? How many filesystems support linktypes really?

Reply:
It's `LocalFileSystem` for now, maybe `SSHFileSystem` in the future.

Reply:
Just need to ignore this particular one for now.

Link types are supported by local and sshfs right now. Might look closer at this again later.

Reply:
> sshfs

is it though? I see https://github.com/fsspec/sshfs/issues/9 still open.

Reply:
Oh, good point. Looks like i forgot that we dropped it. But it is first candidate for more link types. I don't think it is worth messing with it right now, but maybe could consider in the future.